src/components/Toolbelt/Toolbelt.js

```diff
@@ -1,22 +1,33 @@
 import React from 'react'
+import PropTypes from 'prop-types'
+import classNames from 'classnames'
+
 import Button from '@material-ui/core/Button'
 import Tooltip from '@material-ui/core/Tooltip'
-import { func, string } from 'prop-types'
-import classNames from 'classnames'
+
+import { memoize } from '../../utils'
 
 import FarmhandContext from '../../Farmhand.context'
-import './Toolbelt.sass'
-import { tools as toolImages, pixel } from '../../img'
 import toolsData from '../../data/tools'
 
-const tools = Object.values(toolsData).sort(t => t.order)
+import { tools as toolImages, pixel } from '../../img'
+
+import './Toolbelt.sass'
```

Comment:
i added some line breaks to the these imports which added some noise to the PR here